docs/docs/reference/crd-reference/analysisvaluetemplate.md

```diff
@@ -82,6 +75,17 @@ you must define a
 resource for each instance of each data provider you are using.
 The template refers to that provider and queries it.
 
+The Analysis feature uses the
```

Comment:
this paragraph can be removed, as these labels/annotations do not play a role in setting up the Analysis resources

Reply:
Thanks - it's removed.